### PR TITLE
fix spelling error

### PR DESCRIPTION
**Pull request by weizhoublue**
fix spelling error

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/upgrade/_index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Upgrade Istio
 linktitle: Upgrade
-description: Upgrade, downgrade, and manage Istio accross multiple control plane revisions.
+description: Upgrade, downgrade, and manage Istio across multiple control plane revisions.
```
Comment (ericvn) [inline content/en/docs/setup/upgrade/_index.md:4]: This is the only change that is in this repo, so this PR will likely be this one file.

**Review by ericvn — CHANGES_REQUESTED** [review]
Need to remove a number of changes, as they need to be done in other files.

File: content/en/docs/reference/commands/install-cni/index.html [hunk 2]
```diff
@@ -73,7 +73,7 @@
 </tr>
 <tr>
 <td><code>--log-uds-address &lt;string&gt;</code></td>
-<td>The UDS server address which CNI plugin will copy log ouptut to  (default `/var/run/istio-cni/log.sock`)</td>
+<td>The UDS server address which CNI plugin will copy log output to  (default `/var/run/istio-cni/log.sock`)</td>
```
[the remark this answers is not in the document]
Comment (weizhoublue) [reply content/en/docs/reference/commands/install-cni/index.html:76]: got it , thanks for the reminding

**Comment by weizhoublue** [comment]
I will open another PR

**Comment by ericvn** [comment]
I see you created https://github.com/istio/istio/pull/46276 to handle some of the spelling corrections. Are you planning to create the other two PRs based on the changes above? Else I can create the PRs (and make you a co-committer if I know your email).

**Comment by ericvn** [comment]
I found your email so made you a co-commiter on the changes in the other repos. Thanks again for these.

**Comment by weizhoublue** [comment]
> I found your email so made you a co-commiter on the changes in the other repos. Thanks again for these.

@ericvn  sorry for my late reply. I see https://github.com/istio/istio.io/pull/13691 and many thanks

**Pull request closed without merging**